data-model/SDP-data-model-templates/manuscript-object_sdp-template.json

```diff
@@ -65,48 +62,30 @@
         }
       ],
       "sources": [
-        "ARK?"
-      ]
-    }
-  ],
-  "paratexts": [
-    {
-      "id": "ARK",
-      "type": "CV",
-      "locus": "",
-      "transcription": {
-        "$langCode": "Note: lang code refers to the language of transcribed text. transcription object may have multiple $langCode fields."
-      },
-      "translation": {
-        "$langCode": "Note: lang code refers to the language of translated text. translation object may have multiple $langCode fields."
-      },
-      "note": [
-        ""
-      ],
-      "attested_entities": [
-        {
-          "entity_type": "CV",
-          "role": "CV",
-          "id": "URI/ARK",
-          "attested_name_NS": "",
-          "attested_name": "",
-          "note": [
-            "array of strings"
-          ]
-        }
-      ]
-    }
-  ],
-  "associated_entities": [
-    {
-      "entity_type": "CV",
-      "relation": "CV",
-      "id": "URI/ARK",
-      "note": [
-        "array of strings"
+        "ARK/ID of a Paratext? URI of a Zotero Bibl?"
       ]
     }
   ],
+  "associated_entities": {
```

Comment:
I made this a bit more like the related objects field where we have a containing field 'associated_entities' along with more specific fields for each type. I don't think this list is exhaustive necessarily. For example, we could 'attested' as well as 'associated' persons and places. I'm also on the fence about making something like 'attributed_author' be a top-level category at the textual artifact level.